crates/oxc_linter/src/snapshots/no_negation_in_equality_check.snap

```diff
@@ -0,0 +1,93 @@
+---
+source: crates/oxc_linter/src/tester.rs
+---
+  ⚠ eslint-plugin-unicorn(no-negation-in-equality-check): Negated expression is not allowed in equality check.
+   ╭─[no_negation_in_equality_check.tsx:1:1]
+ 1 │ !foo === bar
```

Comment:
Would you like to point out on the negated unary expression, it's where the error occurs.